bowjoin.gen.go.tmpl

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/apache/arrow/go/v8/arrow/array"
 )
 
-func innerFillLeftBowCols(newSeries *[]Series, left, right *bow, newNumRows int,
+func innerFillLeftBowCols(newSeries *[]Series, left *bow, newNumRows int,
```

Comment:
Same question about testing  :)

Reply:
The `right` argument wasn't used in the function at all

Reply:
Ok just if it is impacting an exposed function for the library, this is a breaking change and you need to take it into account

Reply:
Sure, this one is not exposed :)

Reply:
ok for this internal change. Minus "i" in innerFillLeftBowCols means it's not exposed ;) Go minimalism effect.